migration/src/main/resources/META-INF/rewrite/axon-jakarta.yml

```diff
@@ -0,0 +1,74 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.axonframework.axon.UpgradeAxonFramework_4_7_Jakarta
+displayName: Upgrade to Axonframework 4.7 Jakarta
+description: This is what you will want if you also upgrade to Spring 6, Spring Boot 3, or another framework that depends on jakarta packages.
+recipeList:
+  # Convert to Spring Boot 3, Spring FrameWork 6, Jakarta packages, JUnit 5, Java 17, etc.
+  #TODO - org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_0
+  # Swap 4.6 dependencies for 4.7 dependencies
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-configuration-jakarta
+      newArtifactId: axon-configuration
+      newVersion: 4.7.x
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-eventsourcing-jakarta
+      newArtifactId: axon-eventsourcing
+      newVersion: 4.7.x
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-messaging-jakarta
+      newArtifactId: axon-messaging
+      newVersion: 4.7.x
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-modelling-jakarta
+      newArtifactId: axon-modelling
+      newVersion: 4.7.x
+  # Convert legacy JPA packages to Jakarta JPA packages
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.DeadLetterJpaConverter
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.DeadLetterJpaConverter
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.EventMessageDeadLetterJpaConverter
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.EventMessageDeadLetterJpaConverter
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.JpaDeadLetter
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.JpaDeadLetter
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.jpa.JpaSequencedDeadLetterQueue
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.deadletter.legacyjpa.JpaSequencedDeadLetterQueue
+  - org.openrewrite.java.ChangeType:
+      newFullyQualifiedTypeName: org.axonframework.eventhandling.tokenstore.jpa.JpaTokenStore
+      oldFullyQualifiedTypeName: org.axonframework.eventhandling.tokenstore.legacyjpa.JpaTokenStore
+  - org.openrewrite.java.ChangeType:
```

Comment:
We can drop this class. We've made some adjustments in 4.7.1 that allow us to disregard this shift.

With that, I'll adjust the blog and RefGuide accordingly.

Reply:
Removed, assuming there's no chance a user still has that class on their classpath with any version. If they might when they are on 4.7.0 it might still make sense to restore.
```yaml
  - org.openrewrite.java.ChangeType:
      oldFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.legacyjpa.EmbeddedEventStore
      newFullyQualifiedTypeName: org.axonframework.eventsourcing.eventstore.EmbeddedEventStore
```

Reply:
Thanks!